student_explorer/settings.py

```diff
@@ -158,6 +158,8 @@
 
 FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
 
+EMAIL_SUBJECT_PREFIX = 'Student Explorer'
```

Comment:
This should be configurable the same way. Can you make this change and also update the sample .env.sample file documenting this? Just using EMAIL_SUBJECT_PREFIX seems fine to me.

```suggestion
EMAIL_SUBJECT_PREFIX = config('EMAIL_SUBJECT_PREFIX', 'Student Explorer')
```